### DISCUSSION TITLE
Add text directions to the bottom of dungeon embeds

### DISCUSSION THREAD

**Discussion by TPodtepa** [DiscordDungeons/Bugs#954] [Ideas]
To help any blind or visually impaired players out, please consider adding a line of text at the bottom of a dungeon embed that specifically tells you which directions you can go from the current room you are in.

For example, if you're in a central room and can go south and east, have something like

Available paths: east, south

You could even add symbols beside each direction to show what is there, if your character knows/has visited that room previously.

Available paths: north (*) east (?) south (0) west (!)

It would make navigation a lot easier, because as it currently stands I have to count the symbols on each line and then try and figure out what's around. X-axis navigation isn't bad, but Y-axis is more difficult because of the nature of a screen reader to return to the beginning of a line when scrolling up and down, instead of keeping your current position.

**Comment by Sven65** [comment]
I'll add it.
